src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -569,15 +576,6 @@ protected override void ProcessRecord()
                         {
                             long requestContentLength = request.Content is null ? 0 : request.Content.Headers.ContentLength.Value;
 
-                            string reqVerboseMsg = string.Format(
-                                CultureInfo.CurrentCulture,
-                                WebCmdletStrings.WebMethodInvocationVerboseMsg,
-                                request.Version,
-                                request.Method,
-                                requestContentLength);
-
-                            WriteVerbose(reqVerboseMsg);
-
```

Comment:
It is for Verbose. It is another output level. `Invoke-WebRequest -Verbose` vs `Invoke-WebRequest -Debug`.
Please revert.

Reply:
The thing is the data is 100% duplicated between the two, and most people (including myself) find the verbose output here pretty much useless.

Reply:
This is an unrelated change. If it's really useless, then we can discuss it separately and delete it in a separate PR.